### PR TITLE
Pull in upstream updates from cisagov/skeleton-ansible-role

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

In this pull request I pull in the upstream updates from [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role).

## 💭 Motivation and Context

I'm going to be making changes to this repository, so I may as well make sure it is up to date.

## 🧪 Testing

All pre-commit hooks and GitHub Actions pass.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
   platforms:
+    # Kali linux is based on Debian testing
     - name: Debian
       versions:
-        - stretch
-        - buster
-    - name: Amazon
-      versions:
-        - 2018.03
+        - bullseye
```
Comment (dav3r) [inline meta/main.yml:13]: Going straight for the bullseye, eh? 🎯

**Review by dav3r — APPROVED** [review]
🔗

File: meta/main.yml (hunk 1 above)
Comment (jsf9k) [reply meta/main.yml:13]: > Cars are crashing every night
> I drink and drive, everything's in sight
> I make the fire, but I miss the firefight
> I hit the bullseye every night

Comment (felddy) [reply meta/main.yml:13]: 🎶 It's so easy, easy 🎵

**Review by felddy — APPROVED** [review]
🤘

**Review by mcdonnnj — APPROVED** [review]
🏎

**Pull request merged**